Description: Set SQLite as default
 Set SQLite as default db backend
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-07-01

Index: heat/etc/heat/heat.conf.sample
===================================================================
--- heat.orig/etc/heat/heat.conf.sample	2014-10-02 19:01:45.000000000 +0800
+++ heat/etc/heat/heat.conf.sample	2014-10-02 19:01:49.000000000 +0800
@@ -866,7 +866,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=<None>
+connection=sqlite:///var/lib/heat/heatdb
 
 # The SQLAlchemy connection string to use to connect to the
 # slave database. (string value)
